### PR TITLE
chore: improve README

### PR DESCRIPTION
**Pull request by fortuna**
- Add information on how to integrate
- Clean up plan information
- Add image

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,3 +31,152 @@ All submissions, including submissions by project members, require review. We
 use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
+
+# Cross-platform Development
```
Comment (jyyi1) [inline CONTRIBUTING.md:35]: I'm thinking about whether `COMTRIBUTING.md` is the right place to put these instructions. Maybe `devdoc.md` or `devdoc/environment.md` could be a better place?

File: README.md [hunk 2]
```diff
@@ -7,173 +7,61 @@
...
-Tentative roadmap:
+The Outline SDK helps developers:
+- Create tools to protect against network-level interference
+- Add network-level interference protection to existing apps, such as content or communication apps
+
+## Advantages
+
+| Multi-Platform | Proven Technology | Composable |
+|-|-|-|
+| The Outline SDK can be used on Android, iOS, Windows, macOS or Linux. | The Outline Client and Server have been using the SDK for years, helping millions of users in the harshest conditions. | The SDK interfaces were carefully designed to allow for composition and reuse, so you can craft your own transport. |
```
Comment (jyyi1) [inline README.md:25]: > ... have been using the SDK for years ...

Are we gonna really mention "for years" here 😄 ? Because the SDK repository has only been created less than one year.

File: README.md [hunk 3]
```diff
@@ -7,173 +7,61 @@
...
+
+## Integration
+
+The Outline SDK is written in Go. There are multiple ways to integrate the Outline SDK into your app:
+
+- As a **Go library**, in a Go application (CLI or graphical app with frameworks like [Fyne.io](https://fyne.io/), [Wails](https://wails.io/), [Qt for Go](https://therecipe.github.io/qt/), or [Go Mobile app](https://pkg.go.dev/golang.org/x/mobile/app)
+- As a native **mobile library**, using Go Mobile `bind` to generate Java and Objective-C bindings for Android, iOS and macOS.
```
Comment (jyyi1) [inline README.md:32]:
```suggestion
- As a native **mobile library**, using [`gomobile bind`](https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile) to generate Java and Objective-C bindings for Android, iOS and macOS.
```

File: README.md [hunk 4]
```diff
@@ -7,173 +7,61 @@
...
+
+| Multi-Platform | Proven Technology | Composable |
+|-|-|-|
+| The Outline SDK can be used on Android, iOS, Windows, macOS or Linux. | The Outline Client and Server have been using the SDK for years, helping millions of users in the harshest conditions. | The SDK interfaces were carefully designed to allow for composition and reuse, so you can craft your own transport. |
+
+## Integration
+
+The Outline SDK is written in Go. There are multiple ways to integrate the Outline SDK into your app:
+
+- As a **Go library**, in a Go application (CLI or graphical app with frameworks like [Fyne.io](https://fyne.io/), [Wails](https://wails.io/), [Qt for Go](https://therecipe.github.io/qt/), or [Go Mobile app](https://pkg.go.dev/golang.org/x/mobile/app)
```
Comment (jyyi1) [inline README.md]: The Go GUI frameworks doesn't seem to be related to our SDK. Do we need to mention them here?

File: CONTRIBUTING.md (hunk 1 above)
Comment (fortuna) [reply CONTRIBUTING.md:35]: That seems like a good place for someone looking to contribute. I was inspired by Ktor: https://github.com/ktorio/ktor/blob/main/CONTRIBUTING.md

It certainly more discoverable and standard than devdoc.

File: README.md (hunk 2 above)
Comment (fortuna) [reply README.md]: I updated to "The Outline Client and Server have been using the code in the SDK for years" to be clear it's the code, not the SDK

File: README.md (hunk 4 above)
Comment (fortuna) [reply README.md]: People don't know how to do graphical apps in Go, so it's helpful to let them know about some possibilities.
Writing an app on those platforms may be easier than the other integration options.

**Review by jyyi1 — APPROVED** [review]
